notebooks/02- Retail Sales DLT.sql

```diff
@@ -0,0 +1,46 @@
+-- Databricks notebook source
+CREATE INCREMENTAL LIVE TABLE customers
+COMMENT "The customers buying finished products, ingested from /databricks-datasets."
```

Comment:
Add additional comments here using '--' syntax to provide more clues as to what's going on. We want to have well commented code.